apps/token-manager/app/src/components/Panels/AssignVotePanelContent.js

```diff
@@ -93,7 +95,7 @@ class AssignVotePanelContent extends React.Component {
             <TextInput.Number
               value={amount}
               onChange={this.handleAmountChange}
-              min={0}
+              min={minimum}
               step="any"
```

Comment:
Step should also be `minimum`, to avoid people adding decimal precision that will be lost in the contract